Resume.tex

```diff
@@ -55,37 +49,45 @@ \section{Salesforce: June 2015 - current}
     \item Draft example apps demonstrating routing features and customizations, with E2E testing
 \end{itemize}
 
-{\bf Project Manager,} Capstone Projects at Colorado School of Mines
-\begin{itemize} \itemsep -2pt
-    \item Design open-source projects for students to work on as a group (Field Session)
-    \item Lead scrum and keep developers on-task
-    \item DominionStats - An LWC webapp to manage game results with interesting statistics
-    \item UTAM Generator - A static analyzer script that produces instrumented objects for E2E automation
-    \item Agile Accelerator Graph View - A d3.js graph view of related work items
-    \item Story acceptance criteria composer
-    \item Remote programming test manager
-\end{itemize}
-
-{\bf Fullstack Engineer,} Page Loading Optimization \hfill Fall 2018 - Spring 2020
+{\bf Full-stack Engineer,} Page Loading Optimization \hfill Fall 2018 - Fall 2019
 \begin{itemize} \itemsep -2pt
     \item Design and implement framework for organizing data loading based on static analysis
     \item Implement data request bundling
-    \item Patent Application A4968US2\_SFDC061A
+    \item Patent Application 20220236968 - "OPTIMIZED DATA RESOLUTION FOR WEB COMPONENTS"

```

Comment:
The previous appliction number is something internal from the law firm, I think. This one is the findable one via USPTO search